src/functions/template-tags/general.php

```diff
@@ -733,7 +733,7 @@ function tribe_events_get_days_of_week( $format = null ) {
 	 * @return bool
 	 */
 	function tribe_events_admin_show_cost_field() {
-		$modules      = apply_filters( 'tribe_events_tickets_modules', null );
+		$modules      = Tribe__Tickets__Tickets::modules();
```

Comment:
What about deprecating this hook (rather than outright removal) and ensuring it still works, perhaps by wrapping within the `tribe_tickets_get_modules` hook?

Reply:
That gets complicated. The only thing attached to `tribe_events_tickets_modules` is `Tribe__Tickets__Tickets::modules()`. If we ran that filter inside of the attached method, we would face recursion. In order to be fully backwards compatible though the method does need to stay attached to that filter until a non MR.

To account for this I used Tribe__Deprecation within the ET pull request. At this point, if anyone does try to hook into `tribe_events_tickets_modules` they will get a notice telling them its deprecated. I also have a PR for Common now to help facilitate this. https://github.com/moderntribe/tribe-common/pull/521

Reply:
The problem here is that `Tribe__Tickets__Tickets` may not exist when this function executes. It'd only exist if `event-tickets` is active.

Reply:
Removing the filter would probably be pretty low impact, but since the filter is being removed, it warrants a changelog entry and changing the code to the following:

```php
$modules = null;

if ( class_exists( 'Tribe__Tickets__Tickets' ) ) {
  $modules = Tribe__Tickets__Tickets::modules();
}
```

Reply:
I like that.

Reply:
Nice catch. Fixed.